### PR TITLE
Add ebpf backend to CI

### PR DESCRIPTION
**Pull request by astoycos**

### CODE REVIEW

**Comment by astoycos** [comment]
This should be good to go! LMK what you think @knabben

File: hack/test_e2e.sh [hunk 1]
```diff
@@ -1070,7 +1071,11 @@ function main {
     install_binaries "${bin_dir}" "${E2E_K8S_VERSION}" "${OS}"
     # compile bpf bytecode and bindings so build completes successfully
     if [ "${backend}" == "ebpf" ] ; then 
-        compile_bpf 
+        if [ "${ip_family}" != "ipv4" ] ; then 
+            echo "ebpf backend currenlty only supports ipv4"
```
[the remark this answers is not in the document]
Comment (astoycos) [reply hack/test_e2e.sh]: Fixed

**Review by knabben — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**